### PR TITLE
Squash badness

### PR DESCRIPTION
**Pull request by muglug**

### CODE REVIEW

File: src/foo.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 $a = 6;
 
-
 strlen($a);
```
Comment (psalm-spirit) [inline src/foo.php]: Argument 1 of strlen expects string, int(6) provided

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm has thoughts

File: src/foo.php [hunk 2]
```diff
@@ -2,6 +2,7 @@
...
 
+echo $b;
```
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm has thoughts

File: src/foo.php (hunk 1 above)
Comment (psalm-spirit) [inline src/foo.php]: Argument 1 of strlen expects string, int(6) provided

File: src/foo.php (hunk 2 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm found errors in other files

```src/foo.php:5:8 - Argument 1 of strlen expects string, int(6) provided
src/foo.php:7:6 - Cannot find referenced variable $b in global scope```

File: src/foo.php (hunk 1 above)
Comment (psalm-spirit) [inline src/foo.php]: Argument 1 of strlen expects string, int(6) provided

File: src/foo.php (hunk 2 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm found some errors

File: src/foo.php (hunk 1 above)
Comment (psalm-spirit) [inline src/foo.php]: Argument 1 of strlen expects string, int(6) provided

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm found some errors

File: src/foo.php (hunk 1 above)
Comment (psalm-spirit) [inline src/foo.php]: Argument 1 of strlen expects string, int(6) provided

File: src/foo.php (hunk 2 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm didn’t find any errors!

**Pull request merged**